openreview/arr/webfield/areachairsWebfield.js

```diff
@@ -70,7 +71,7 @@ return {
       showEdgeBrowserUrl: domain.content.enable_reviewers_reassignment?.value,
       proposedAssignmentTitle: reviewerAssignmentTitle,
       edgeBrowserProposedUrl: `/edges/browse?start=${startParam}&traverse=${traverseProposedParam}&edit=${replaceReviewerName(domain.content.reviewers_proposed_assignment_id?.value)},label:${reviewerAssignmentTitle};${replaceReviewerName(domain.content.reviewers_invite_assignment_id?.value)}&browse=${browseProposedInvitations.join(';')}${otherParams}`,
-      edgeBrowserDeployedUrl: `/edges/browse?start=${startParam}&traverse=${traverseParam}&edit=${replaceReviewerName(domain.content.reviewers_invite_assignment_id?.value)};${replaceReviewerName(domain.content.reviewers_assignment_id?.value)}&browse=${browseInvitations.join(';')}${otherParams}`,
+      edgeBrowserDeployedUrl: `/edges/browse?start=${startParam}&traverse=${traverseParam}&edit=${replaceReviewerName(domain.content.reviewers_invite_assignment_id?.value)}&browse=${browseInvitations.join(';')}${otherParams}`,
```

Comment:
I took the changes from the December version and there is a difference here, can you check what is correct?

Reply:
Oh we should leave this in here `;${replaceReviewerName(domain.content.reviewers_assignment_id?.value)}` the ACs have been allowed to use both Invite Assignment and Assignment since API1

Reply:
do you know why December doesn't have this param?

Reply:
Hmm I'm not sure I'm taking a look at the group revisions now

Reply:
I'm not sure how I missed this but it looks like the AC webfield isn't being set in the ARR setup function, only the other webfields are
https://github.com/openreview/openreview-py/blob/845523278d696b593bd6f715cecc915ee82a753a/openreview/arr/arr.py#L348

Reply:
how December has the webfield located in the ARR repo?

Reply:
It looks like it does because it's getting changes from this process function (date process function run from a date in the ARR Configuration): https://github.com/openreview/openreview-py/blob/master/openreview/arr/management/setup_proposed_assignments.py
I originally did it this way because there were many places in API1 where we did this modification of the webfield

It seems like when we added the AC webfield we forgot to add the logic